exporter/exporterhelper/internal/base_exporter.go

```diff
@@ -275,30 +275,14 @@ func WithCapabilities(capabilities consumer.Capabilities) Option {
 	}
 }
 
-// WithRequestBatchFuncs sets the functions for merging and splitting batches for an exporter built for custom request types.
-func WithRequestBatchFuncs(mf exporterbatcher.BatchMergeFunc[internal.Request], msf exporterbatcher.BatchMergeSplitFunc[internal.Request]) BatcherOption {
```

Comment:
This is a significant cut of the functionality. There is no longer any way to use batching for custom requests. Are we planning to bring it back later? Can we make `WithBatchFuncs` exported now as an alternative?

Reply:
> Are we planning to bring it back later? 

I think we will do by adding it to the `internal.Request`.

Reply:
By adding required methods or as an optional interface?

Reply:
> By adding required methods or as an optional interface?
It should be optional because batching is not necessarily enabled.

Reply:
They will be optional for the final user to implement. To instantiate the Batcher they will be required.

Reply:
Ok, but for now, can we just export `WithBatchFuncs` to not remove the capability? Something
```
func WithBatchFuncs(mf exporterbatcher.BatchMergeFunc[Request], msf exporterbatcher.BatchMergeSplitFunc[Request]) Option {
	return internal.WithBatchFuncs(mf, msf)
}
```

Reply:
I am confused, isn't the case that nobody should use this because `Request` is still internal?

Reply:
Ignore, is exported in the helper.